backend/business-partner-agent/src/main/java/org/hyperledger/bpa/api/aries/SchemaAPI.java

```diff
@@ -54,11 +55,21 @@
 
     private List<CredDef> credentialDefinitions;
 
+    private Boolean isCreator;
+
     public static SchemaAPI from(BPASchema s) {
-        return from(s, true, true);
+        return from(s, true, true, null);
+    }
+
+    public static SchemaAPI from(BPASchema s, String creatorDid) {
+        return from(s, true, true, creatorDid);
     }
 
     public static SchemaAPI from(BPASchema s, boolean includeRestrictions, boolean includeCredDefs) {
```

Comment:
just as a remark. If model construction gets to complex I would propose views (@JsonView) or introduce different view models, then you can ask the converter what to return instead of putting everything into one place.